src/layman/upgrade/upgrade_v1_22_test.py

```diff
@@ -33,3 +33,41 @@ def test_fix_issuer_id():
         issuer_id_rows = db_util.run_query(query)
     assert len(issuer_id_rows) == 1
     assert issuer_id_rows[0][0] == 'layman.authn.oauth2'
+
+
+@pytest.mark.usefixtures('ensure_layman')
+def test_insert_map_layer_relations():
```

Comment:
Franta wants to import also layers with client-proxy prefix in URL.
- We can add test case now with skip and remove it later (what about burn-down chart)?
- We can implement this functionality to get_layers_from_json in this PR.